packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -7,6 +7,8 @@ export interface DropdownItemProps extends InternalDropdownItemProps {
   children?: React.ReactNode;
   /** Classes applied to root element of dropdown item */
   className?: string;
+  /** Class to be applied to list item */
+  wrapperClass?: string;
```

Comment:
Perhaps this could be named something more specific, like `listItemClassName` or `itemClassName`? In case we need other class names in the future.